src/routes/workspace/sidebar/quickActions/QuickTabs.svelte

```diff
@@ -50,7 +45,7 @@
 			<ClipboardList size="27"></ClipboardList>
 		</QuickTab>
 	{/if}
-	{#if $currentFile instanceof AssignmentAnswer || ($currentFile instanceof Assignment && $currentFile.isPublic && $answer)}
+	{#if showAssignmentTabs}
```

Comment:
The feedback tab should only be visible if the teacher is currently looking at an assignmentAnswer. i.e. $answer